src/components/settings/SettingsLMSTab/LMSConfigPage.jsx

```diff
@@ -35,7 +35,7 @@ export function handleErrors(error) {
     ? error.message
     : JSON.stringify(error.response.data);
   logError(errorMsg);
-  return errorMsg;
+  return error.response?.status;
```

Comment:
there are probably situations where there is no response - perhaps a timeout, dropped connection, browser extension blocking. will consumers of `handleError` be okay if this returns a `null`?